src/private/conversations.ts

```diff
@@ -23,9 +23,12 @@ export namespace conversations {
       subEntityId: openConversationRequest.subEntityId,
       conversationId: openConversationRequest.conversationId
     }]);
-    GlobalVars.getConversationIdHandler = openConversationRequest.onCloseConversation;
-    GlobalVars.callbacks[messageId] = (conversationId?: string) => {
-      openConversationRequest.onStartConversation(conversationId);
+    GlobalVars.onCloseConversationHandler = openConversationRequest.onCloseConversation;
+    GlobalVars.onStartConversationHandler = openConversationRequest.onStartConversation;
```

Comment:
Should we at some point clear out these handlers? Maybe when the closeConversation event comes in?

Reply:
agree with @ydogandjiev